clippy_utils/src/msrvs.rs

```diff
@@ -73,7 +73,7 @@ msrv_aliases! {
     1,29,0 { ITER_FLATTEN }
     1,28,0 { FROM_BOOL, REPEAT_WITH, SLICE_FROM_REF }
     1,27,0 { ITERATOR_TRY_FOLD }
-    1,26,0 { RANGE_INCLUSIVE, STRING_RETAIN }
+    1,26,0 { RANGE_INCLUSIVE, STRING_RETAIN, POINTER_METHODS }
```

Comment:
This is a bit too general.


```suggestion
    1,26,0 { RANGE_INCLUSIVE, STRING_RETAIN, POINTER_ADD_METHODS }
```